lib/components/table.vue

```diff
@@ -148,7 +148,7 @@ function recToString(obj) {
 
     return toString(keys(obj).reduce((o, k) => {
         // Ignore fields that start with _
-        if (!/^_/.test(k)) {
+        if (!'/^_/'.test(k)) {
```

Comment:
Can't have quotes around a literal regex

Reply:
Yep, my bad english...  eslint recommended parens not quotes ))